.ci/azure/linux_debian.yml

```diff
@@ -327,7 +328,7 @@ jobs:
 
     # Skip test_onnx/test_zoo_models and test_onnx/test_backend due to long execution time
   - script: |
-      python3 -m pytest -s $(INSTALL_TEST_DIR)/pyngraph \
+      /usr/bin/python3 -m pytest -s $(INSTALL_TEST_DIR)/pyngraph \
```

Comment:
$(TEST_VENV) ?

Reply:
no, I explicitly want to test python API installed via `apt-get install python3-openvino`